src/fixers/misc_tracker.ts

```diff
@@ -0,0 +1,348 @@
+
+            if (travelColumn.querySelector("div[class='snippet']")) {
+                // Give some clearance in case snippets are not disabled.
+                (sidebar as HTMLElement).style.cssText = "margin-top: 30px";
+            }
+        }
+
+        let miscPanel = document.getElementById("misc-tracker");
+        // Trackers are already created and visible, nothing to do here.
+        if (!miscPanel) {
+            const fragment = document.createDocumentFragment();
+
+            const miscHeader = document.createElement("p");
+            miscHeader.classList.add("heading", "heading--4");
+            miscHeader.textContent = "Tracked Values";
+            fragment.appendChild(miscHeader);
+
+            miscPanel = document.createElement("ul");
+            miscPanel.setAttribute("id", "misc-tracker");
+            miscPanel.classList.add("items", "items--list");
+            fragment.appendChild(miscPanel);
+
+            for (const valueName of this.miscQualities.keys()) {
+                const miscDisplay = this.createTracker(valueName);
+                miscPanel.appendChild(miscDisplay);
+            }
+
+            const miscPicker = document.createElement("div");
+            const miscSelect = document.createElement("select");
+            miscSelect.id = "track-target-name";
+            miscPicker.appendChild(miscSelect);
+            for (const qualityName of this.qualityNames) {
+                const option = document.createElement("option");
+                option.value = qualityName;
+                option.text = qualityName;
+                miscSelect.appendChild(option);
+            }
+            const targetInput = document.createElement("input");
+            targetInput.id = "track-target-number";
+            targetInput.type = "number";
+            miscPicker.appendChild(targetInput);
+
+            const trackButton = document.createElement("button");
+            trackButton.classList.add("js-tt", "button", "button--primary", "button--margin", "button--go")
+            trackButton.addEventListener("click", () => {
+                const trackName = (document.getElementById("track-target-name") as HTMLSelectElement).value;
+                const trackNumber = Number((document.getElementById("track-target-number") as HTMLInputElement).value);
+                const trackCategory = this.qualityNameAndCategory.get(trackName) || "";
+                const trackCurrent: number = this.currentState?.getQuality(trackCategory, trackName)?.level || 0;
+                const trackImage: string = this.currentState?.getQuality(trackCategory, trackName)?.image || "question"
+                const newQuality = { name: trackName, category: trackCategory, currentValue: trackCurrent, targetValue: trackNumber, image: trackImage };
+                console.log("before add")
+                console.log(this.miscQualities)
+                console.log(this.currentSettings)
```

Comment:
If you output debug information, then please use the appropriate `debug()` function supplied by the extension itself. It will both set the appropriate reporting level and append the prefix annotating that the message belongs to this extension.

Reply:
Woops, these should have all been removed once I got it working